src/backend/marsha/core/models/video.py

```diff
@@ -200,6 +200,7 @@ class Video(BaseModel):
         choices=STATE_CHOICES,
         default=PENDING,
     )
+    is_downloadable = models.BooleanField(default=True)
```

Comment:
I thought of putting this flag on the playlist so that all videos/tracks of a course get the same choice. It's less of a hassle for the teacher? And do you think a teacher could want some videos to be downloadable and some not in the same course ?

Reply:
I discussed about this with Cédric. In video front this parameter is on each video. It could be great if this parameter is for the entire course. **but** this parameter could be overrided at the video level. An instructor could decide to flag this property to false at the course level and then flag it to true in a specific video. And I have no idea how to deal with this constraint.

Reply:
Abd the playlist can be share with another course ? If yes you have an other use case to manage.

Reply:
No such problem because playlist == LTI `context_id` == course.